### PR TITLE
feat(cf-component-dynamic-content): css2fela

### PR DESCRIPTION
**Pull request by koddsson**
Convert the `cf-component-dynamic-content` to fela.

Related-to: #227

### CODE REVIEW

File: test_setup/patch-console.js [hunk 1]
```diff
@@ -10,6 +10,8 @@ global.console.trace = () => {
   throw new Error("Please don't call console.trace in the codebase");
 };
 
-global.console.log = () => {
+const log = global.console.log;
+global.console.log = message => {
+  log(message);
```
Comment (koddsson) [inline test_setup/patch-console.js]: I added this here so I can add console logs temporarily when debugging but still have this fail.

File: packages/cf-component-dynamic-content/src/DynamicContent.js [hunk 2]
```diff
@@ -1,12 +1,16 @@
 import React from 'react';
-
 import PropTypes from 'prop-types';
+import { createComponent } from 'cf-style-container';
+
+const styles = ({ theme }) => ({
+  display: theme.display
```
Comment (hturan) [inline packages/cf-component-dynamic-content/src/DynamicContent.js]: It looks like we override `color` and `textDecoration` internally. Will we still be able to do so here?

File: packages/cf-component-dynamic-content/src/DynamicContentTheme.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export default baseTheme => ({
+  display: 'initial'
```
Comment (sejoker) [inline packages/cf-component-dynamic-content/src/DynamicContentTheme.js]: `initial` is the default  value, safe to remove

File: packages/cf-component-dynamic-content/src/DynamicContent.js (hunk 2 above)
Comment (koddsson) [inline packages/cf-component-dynamic-content/src/DynamicContent.js]: We talked about this IRL

File: packages/cf-component-dynamic-content/src/DynamicContentTheme.js (hunk 3 above)
Comment (koddsson) [inline packages/cf-component-dynamic-content/src/DynamicContentTheme.js]: I needed here if I want to override it.

**Pull request merged**